### PR TITLE
DetailView - List to display Client informations

### PR DESCRIPTION
**Pull request by palant-dev**
Created a new view with dummy UI with informations about client.

### CODE REVIEW

**Comment by palant-dev** [comment]
client view

File: SlimApp/View/ClientDetailsView.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    var body: some View {
+        VStack {
+            
+            Circle()
+                .foregroundColor(.gray)
+                .frame(width: 200, alignment: .center)
+                .overlay {
+                    Image(systemName: "person.fill")
+                        .foregroundColor(.white)
+                        .font(.custom("sf-arabic", fixedSize: 150))
```
Comment (giiks131) [inline SlimApp/View/ClientDetailsView.swift]: what is the reason for this arabic custom font?

File: SlimApp/View/ClientDetailsView.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                }
+            
+            List {
```
Comment (giiks131) [inline SlimApp/View/ClientDetailsView.swift]: I think it is better to use Form in such case or customize List. The reason is that i don't think that we have a need to hide our sections under the title

File: SlimApp/View/ClientDetailsView.swift [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                } header: {
+                    Text("Diseases")
+                }
+                
+                Section {
+                    Text("Lorem ipsum")
+                } header: {
+                    Text("Notes")
+                }
+
```
Comment (giiks131) [inline SlimApp/View/ClientDetailsView.swift]: better to clear this whitespaces

**Review by giiks131 — COMMENTED** [review]
review for clientDataView

File: SlimApp/View/ClientDetailsView.swift (hunk 1 above)
Comment (apalomba22) [reply SlimApp/View/ClientDetailsView.swift:20]: required argument for .custom

File: SlimApp/View/ClientDetailsView.swift (hunk 3 above)
Comment (palant-dev) [reply SlimApp/View/ClientDetailsView.swift]: Done!

File: SlimApp/View/ClientDetailsView.swift (hunk 2 above)
Comment (palant-dev) [reply SlimApp/View/ClientDetailsView.swift]: I analysed the problem, and decided to keep the functionalities that List gives us while hiding the chevron button for hiding/showing sections with `.listStyle(.insetGrouped)`

File: SlimApp/View/ClientDetailsView.swift (hunk 1 above)
Comment (palant-dev) [reply SlimApp/View/ClientDetailsView.swift]: Edited with a different modifier: `.font(.system(size: 150))`

**Pull request merged**